dev/ci/ci-basic-overlay.sh

```diff
@@ -306,7 +306,7 @@
 # menhirlib
 ########################################################################
 # Note: menhirlib is now in subfolder coq-menhirlib of menhir
-: "${menhirlib_CI_REF:=master}"
+: "${menhirlib_CI_REF:=20201122}"
```

Comment:
It doesn't work with `master`?

Reply:
no, compcert wants a date grated than... a dev version makes configure fail